Source/Core/DolphinWX/Config/AudioConfigPane.h

```diff
@@ -50,4 +51,6 @@ class AudioConfigPane final : public wxPanel
   wxStaticText* m_stretch_label;
   DolphinSlider* m_stretch_slider;
   wxStaticText* m_stretch_text;
+
+  bool m_any_backend_with_latency_control;
```

Comment:
How do you feel about renaming this to m_latency_control_supported?

Reply:
Better, done.